blink/README.md

```diff
@@ -2,19 +2,31 @@
 
 ## Overview
 
-The [Blink][1] integration is a collection of no-code actions and triggers usable in Blink Automations. Instead of writing manual API calls and scripts, you can use Blink's visual editor to drag-and-drop Datadog API actions into your workflow or use one of the hundreds of existing pre-made automations from the [library][2].
+[Blink][1] is a low-code/no-code (LNCN) platform that enables automated incident response, cloud-native operations, and security operations workflows. Blink transforms manual tasks into interactive automations backed by the security and reliability of a cloud-native platform. Every script or ticket becomes a fully-managed automation.
 
-This integration enables you to:
-1. Trigger automations within Blink in response to new Datadog incidents 
-2. Create and update Datadog incidents in Blink 
-3. View incidents or events from the Datadog event stream in Blink
+The user interface and [automation library][2] come with over 5,000 cloud-native workflows that make it easy to create new automations. Blink helps you achieve better cloud efficiency and more competitive SLA's, with fewer operational bottlenecks.
```

Comment:
Is this associated to Datadog? We try to avoid marketing material for integrations

Reply:
It isn't directly associated to Datadog. To link it more to the integration, we can possibly change it to be a link to premade Datadog automations to show examples of how to use the two platforms.